packages/core/src/login/webview/vue/login.vue

```diff
@@ -378,6 +390,13 @@ export default defineComponent({
                     connectionId: connection.id,
                 })
             })
+            // fetch existing connections of itself
+            const connections = await client.listConnections()
+            connections.forEach(connection => {
+                if ('startUrl' in connection) {
+                    this.existingConnectionStartUrls.push(connection.startUrl)
```

Comment:
how does this cache of URLs get updated if the user logs out?

Reply:
Every time when the webview is rendered, it gets updated.